src/fides/core/export_helpers.py

```diff
@@ -350,7 +351,6 @@ def union_data_categories_in_joined_dataframe(joined_df: pd.DataFrame) -> pd.Dat
     # isolate the system data categories into a new dataframe and create a common column
     joined_df = joined_df.drop(
         [
-            "system.privacy_declaration.name",
```

Comment:
I noticed that you're tagged with the git blame for this line. Do you know why this field was being dropped? @SteveDMurphy

Reply:
i certainly don't have the necessary context, but i did notice in the datamap's system configuration UI, the `Privacy declaration name` field is listed as "deprecated" - so maybe that relates to why it was being dropped on the response here?

![image](https://user-images.githubusercontent.com/9750285/225330871-25fefbb0-b217-487c-9772-d0dd8b37b10a.png)

Reply:
Sorry for the holdup @TheAndrewJackson ! Just looking but initial thoughts are that I believe it would have caused duplicate rows if the same data category/use/subject were used commonly across privacy declarations. 

I don't know that it was actually a requirement of what was originally on the datamap so it may have been dropped to remove duplication (which was an early problem with the datamap). 

Adding it back in will likely have a potential for further duplication in those scenarios

Reply:
I see. Do you think it's too risky to merge this? We can always roll the change back if it causes issues. 

This column will probably be removed again once the customer has migrated away from using it.